.gitmodules

```diff
@@ -7,8 +7,7 @@
 [submodule "third_party/grpc"]
 	path = third_party/grpc
 	url = https://github.com/grpc/grpc.git
-	branch = v1.6.x
-	ignore = dirty
```

Comment:
Is this no longer necessary? I thought this directory is modified during the build, which is why we had this line.

Reply:
Yup.  I forgot to restore it.  Annoyingly, it is hard to see what you are doing when updating grpc if the line is there.